ui/src/app/pages/admin/user/admin-user-table.tsx

```diff
@@ -59,6 +59,17 @@ const DisabledIcon = () => (
   </span>
 );
 
+const Temp = () => (
```

Comment:
Not sure if the question is still open, but my hypothesis would be:

DataTable looks at its children to determine the column schema.
Specifically, it probably only considers children of type `Column`, or else only children which have the props it expects.
Adding this Temp component in the middle would likely break that interaction

I suspect this would work:

```
const temp = () => {
...
}

...
{temp()}
<Column ...
```

I don't know if there's another way to eliminate `Temp` from the component hierarchy while still using it like a Component, it seems like that might break a lot of fundamental rules of React.

Reply:
Interesting, thanks.  This approach does actually work.

Reply:
https://github.com/all-of-us/workbench/pull/6703